includes/class-sensei-updates.php

```diff
@@ -81,11 +85,32 @@ public function run_updates() {
 		$this->v3_9_fix_question_author();
 		$this->v3_9_remove_abandoned_multiple_question();
 		$this->v4_10_update_install_time();
+		$this->create_default_emails();
```

Comment:
Should we keep the version prefix convention?

Reply:
I thought about moving it to Installer later (when we merge it to trunk).
It looks like a very general part of installation.
However, we can add that prefix for consistency until then.